i18n/nitrokey_pl.ts

```diff
@@ -1150,7 +1146,7 @@ Zmień kod PIN.</translation>
     </message>
     <message>
         <source>Enter Firmware Password:</source>
-        <translation>Wprowadź hasło do firmware&apos;u</translation>
+        <translation>Wprowadź hasło do firmware'u</translation>
```

Comment:
`'` characters might be automatically converted to a HTML entity during strings update, like in this case to `&apos;` - please ignore in the future. No need to correct this one back.

Reply:
Sure. I will need detailed explanation what "Password Safe" is doing --> first option on the left side next to "unlock encrypted volume"

Reply:
Great! 
PWS is a hardware password vault, where one can store/read logins and passwords to internet sites or other resources.